pkg/bucket/bucket.go

```diff
@@ -117,11 +117,12 @@ func RunUpdate(cmd *cobra.Command, args []string) {
 	bucketName := args[0]
 	nbClient := system.GetNBClient()
 
-	forceMd5Etag, _ := cmd.Flags().GetBool("force_md5_etag")
-
-	err := nbClient.UpdateBucketAPI(nb.CreateBucketParams{Name: bucketName, ForceMd5Etag: forceMd5Etag})
-	if err != nil {
-		log.Fatal(err)
+	if !cmd.Flags().Changed("force_md5_etag") {
```

Comment:
shouldn't it be without the `!`?

```suggestion
	if cmd.Flags().Changed("force_md5_etag") {
```

Reply:
Updated, thanks.